coalib/bearlib/languages/documentation/DocumentationComment.py

```diff
@@ -204,3 +204,22 @@ def from_metadata(cls, doccomment, docstyle_definition,
 
         return DocumentationComment(assembled_doc, docstyle_definition, indent,
                                     marker, range)
+
+    def assemble(self):
+        """
+        Assembles parsed documentation to the original documentation.
+
+        This function assembles the whole documentation comment, with the
+        given markers and indentation.
+        """
+        lines = self.documentation.splitlines(keepends=True)
+        assembled = ((self.indent if lines[0][0] == '\n' else '') +
+                     self.marker[0])
```

Comment:
not just `self.indent + self.marker[0]` as the extracted documentation starts after the first marker (which can't be `''`)?